requirements.txt

```diff
@@ -1,3 +1,4 @@
+bayescd==0.4
```

Comment:
We should change this to >= 0.4 
Right now I see 0.4 is the most recent version.

Reply:
Fixed!